.github/workflows/iroha2-dev-pr.yml

```
@@ -44,24 +48,28 @@ jobs:
     container:
       image: 7272721/i2-ci:latest
     steps:
+      - uses: actions/checkout@v2
+
       # TODO Remove these steps #2165
       - uses: actions-rs/toolchain@v1
         with:
           toolchain: stable
           override: true
           components: llvm-tools-preview
       - uses: taiki-e/install-action@cargo-llvm-cov
```

Please keep the range of steps clear in some way

```suggestion
      # TODO Remove these steps #2165
      - uses: actions-rs/toolchain@v1
        with:
          toolchain: stable
          override: true
          components: llvm-tools-preview
      - uses: taiki-e/install-action@cargo-llvm-cov
      #
```